css/styles.css

```diff
@@ -16,7 +16,6 @@
 
 body{
     width: 500px;
-    /* use dyslexic friendly background */
```

Comment:
Could you keep this comment? This one helps get an idea of what is happening

Reply:
Sir thank you for suggesting, completed sir